virttest/utils_test/libvirt.py

```diff
@@ -2327,6 +2328,8 @@ def create_tpm_dev(params):
         tpm_backend.backend_type = backend_type
         if backend_version:
             tpm_backend.backend_version = backend_version
+        if persistent_state:
```

Comment:
According to line 2320, persistent_state won't be 'False' unless being assigned with 'False', empty str/list/dict....  
So basically this line is always True, right?

Reply:
Modified. Thanks.